SEED-platform/seed
# Inventory View Fixes

#### What's this PR do?
- Fixed a bug where extra_data columns weren't returned for related taxlot records
- Fixed the show-populated-columns modal to request all columns to properly filter them
- Restored property/taxlot "list" resources that now return all columns for all records
- Passing -1 as the profile id when requesting inventory returns all columns
- Also fixed csv line endings

#### How should this be manually tested?
- Try out the `Only Show Populated Columns` dialog.  Make sure that it shows the expected amount of columns for both properties and taxlots, and check that extra data columns are actually populated in the inventory list view

#### What are the relevant tickets?
#1601
